Rethinking/end-of-chapter-practice-problems/ch-2.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>Small mix-up in the sentence up there. </p><p>Proposal : "A probabilistically equivalent statement is to pretend there are one B/B, two B/W and three W/W card in the bag"</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>